numpy/ma/tests/test_core.py

```diff
@@ -3186,6 +3186,96 @@ def test_empty(self):
         b = a.view(masked_array)
         assert_(np.may_share_memory(a.mask, b.mask))
 
+    def test_zeros(self):
```

Comment:
The new tests here seem fine, but I would prefer either that the `x` and `x_like` tests were either broken up into to separate tests (e.g. `test_zeros` and `test_zeros_like`) or that the test names be amended to encompass both (e.g. `test_zeros_and_zeros_like`)